.github/workflows/build_and_test.yml

```diff
@@ -80,15 +80,17 @@ jobs:
 
   wasm-build:
     name: run wasm build script
-    runs-on: ubuntu-latest
     strategy:
       matrix:
-        toolchain:
-          - stable
+        os: [ubuntu-latest, macos-latest-large, macos-latest]
```

Comment:
Good idea to test this 🙂 What's the reason for running on both `macos-latest` and `macos-latest-large`?

Reply:
One is Intel and the other is arm64. Referece: https://github.com/actions/runner-images?tab=readme-ov-file#available-images